incubator/nodejs-express/README.md

```diff
@@ -1,79 +1,65 @@
+Templates are used to create your local project and start your development. When initializing your project you will be provided with the simplest Node.js Express application you can write.
 
-### simple template
+This template only has a simple `app.js` file which implements the `/` endpoint. The application metadata is provided via a package.json file.
 
-This is the simplest node.js express application you can write. This template only has a simple `app.js` file which implements the `/` endpoint.
+## Getting Started
 
-### skaffold template
+1. Create a new folder in your local directory and initialize it using the Appsody CLI, e.g.:
 
-This is typical skaffolding for a basic node.js express application. This template introduces the `routes` and `views` folders, to organize various components of your appplication and has implementation for the `/` endpoint.
+    ```bash
+    mkdir my-project
+    cd my-project
+    appsody init nodejs-express
+    ```
 
-## Getting Started
+    This will initialize a Node.js Express project using the default template.
+
+1. After your project has been initialized you can then run your application using the following command:
+
+    ```bash
+    appsody run
+    ```
+
+    This launches a Docker container that continuously re-builds and re-runs your project, exposing it on port 3000.
+
+    You can continue to edit the application in your preferred IDE (VSCode or others) and your changes will be reflected in the running container within a few seconds.
+
+1. You should be able to access the following endpoints that are exposed by your template application by default:
 
-### Initializing the project
-
-1. To start using this Node.js Express stack you can create a new folder in your local directory and initialize it using with the Appsody CLI eg:
-```bash
-mkdir my-project
-cd my-project
-appsody init nodejs-express
-```
-This command creates a new project based on the Node.js Express hello-world Template and then installs your development environment. This will also also install all your applications dependancies.
-
-2. After your project has been intialized you can then run your application using the following command:
-```bash
-appsody run
-```
-This launches a Docker container that continuously re-builds and re-runs your project, exposing it on port 3000.
-
-Note: you can continue to edit the application in VSCode IDE. Changes will be reflected in the running container around 2 seconds after the changes are saved.
-
-# Pre-release
-
-1. Cloning Appsody stacks repository
-```bash
-git clone https://github.com/appsody/stacks
-```
-2. Building the nodejs-express stack
-```bash
-cd incubator/nodejs-express/stack
-docker build -t nodejs-express -f Dockerfile-stack .
-```
-3. Run your application
-```bash
-appsody run
-```
\ No newline at end of file
+Health endpoint: http://localhost:3000/health
```

Comment:
These are displaying horizontally across the page not in the list. Also can you put the Application endpoint first, as we didn't explicitly put this in the Getting Started?

Reply:
sorted formatting and order

Reply:
resolved